src/components/eRACommons.js

```diff
@@ -56,9 +54,8 @@ export const eRACommons = hh(class eRACommons extends React.Component {
     }
   };
 
-  getResearcherProperties = async () => {
-    const currentUserId = Storage.getCurrentUser().dacUserId;
-    Researcher.getPropertiesByResearcherId(currentUserId).then(response => {
+  getUserInfo = async () => {
+    User.getMe().then((response) => {
       const isAuthorized = JSON.parse(_.get(response, 'eraAuthorized', "false"));
       const expirationCount = AuthenticateNIH.expirationCount(_.get(response, 'eraExpiration', 0));
       const nihValid = isAuthorized && expirationCount > 0;
```

Comment:
Here, again, the responses are different between the two API calls. I'm pretty sure that `_.get(response, 'eraAuthorized', "false"))` will always be false since we don't store `eraAuthorized` directly on the user.